prisma/datamodel.prisma

```diff
@@ -52,4 +53,13 @@ type Event {
     image_url: String!
     description: String
     subscribers: [User] @relation(name: "EventSubscribers")
+    comments: [Comment] @relation(name: "EventComments", onDelete: CASCADE)
+}
+
+type Comment {
+    id: ID! @id
+    content: String!
+    user: User! @relation(name: "UserComments")
+    event: Event! @relation(name: "EventComments")
```

Comment:
I wonder if we could reuse `Comment`s in other applications in which case, it might not be tied to a particular `Event`. So maybe `event: Event` (without `!` non-null requirement) is a better idea?

Reply:
In this case, the `crud` methods for `Comment` should not be restricted to just for an `Event`

Reply:
ohh that's a really good input. I'll push some commits in regard to that